### ISSUE TITLE
Mention users

### ISSUE THREAD

**Issue by driesvints** [laravelio/laravel.io#271]
People should be able to mention users in replies and threads with an `@` tag. Upon thread or reply creation, these tags are detected and the user mentioned gets an email notification. This should only happen for thread or reply creation and not with updating them. User mentions in threads and replies should be indicated with a link to their profile and be styled as bold.

It's also important that we provide a way for users to unsubscribe from notifications on a thread. Atm a reply to a thread they're subscribed to already sends a notification. 

So basically if a user is mentioned in a thread:

1. and they aren't subscribed yet to the thread, they're being subscribed
2. they receive a specific notification/email for the mention: "X has mentioned you in a thread"
3. they can unsubscribe again from the email in the same way the current reply-notification offers

We should probably also think of a way to prevent abuse. GitHub implements this as an opportunity to block a user. But that's for a different solution.

**Comment by wilburpowery** [comment]
Anyone working on this? @driesvints, I can give it a shot.

**Comment by driesvints** [comment]
@wilburpowery I'm currently working on implementing subscriptions. I guess when that's done this can be picked up.

**Comment by Luciad71** [comment]
> @wilburpowery I'm currently working on implementing subscriptions [url=https://www.comparateur-mutuelle-assurance-sante.com/mutuelle-travailleur-independant]mutuelles indépendants[/url]. I guess when that's done this can be picked up.
Obviamente, si ya se ha hecho, debe recuperarse, pero espere hasta que se estabilice primero.

**Comment by pokurivijay** [comment]
Any progress on this feature?

**Comment by driesvints** [comment]
Sent in a tiny part of this in here: https://github.com/laravelio/laravel.io/pull/643
